src/magento/magento-account-share.md

```diff
@@ -3,57 +3,96 @@ title: Sharing Your Account
 group: getting-started
 ---
 
-Your Magento account contains information that can be useful to trusted employees and service providers who help to manage your site. As the primary account holder, you have authority to grant limited access to your account to other Magento account holders. When your account is shared, all sensitive information—such as your billing history or credit card information—remains protected. It is not shared at any time with other users.
+Your Magento account contains information that you can make available to trusted employees and service providers who help manage your site. As the primary account holder, you have authority to grant limited access to other Magento account holders. Shared access can be revoked, but cannot be transferred from one user to another.
+
+The Magento Support team does not have access to the account and cannot set up shared access for you. Only the primary account holder with appropriate [permissions]({% link system/permissions.md %}) can set up shared access. When your account is shared, all sensitive information &#8212; such as your billing history or credit card information &#8212; remains protected and is not shared at any time with other users.
 
 {:.bs-callout-info}
-All actions taken by users with shared access to your account are your sole responsibility. Magento Inc. is not responsible for any actions taken by users to whom you grant shared account access.
+All actions taken by users with shared access are the sole responsibility of the primary account holder. Adobe is not responsible for any actions taken by users who have shared access to your account.
 
-![]({% link images/images/magento-account-shared.png %}){: .zoom}
+![]({% link images/images/magento-account-shared-access.png %}){: .zoom}
 _Shared Access_
 
-## To set up a shared account:
+## Set up a shared account
 
 1. Before you begin, get the following information from the Magento account of the **new user**:
```

Comment:
not sure why "new user" is bolded here...

Reply:
This update  shouldn't be included in this branch -- I didn't catch it.